Bottom: 9e2767aaa557e631a6ae810302a72a2396153fb8
Top:    f9dbc4b10a50542577c49473c0ec41fc71abad70
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 15:14:15 +0100

Refresh of doc-remove-eps-files-in-favour

---

diff --git a/doc/Makefile.am b/doc/Makefile.am
index d6e6543..1e7d430 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -60,6 +60,7 @@ mostlyclean-aminfo:
 
 EXTRA_DIST= \
 	ascii2texi.awk \
+	eps2png \
 	extract-docs \
 	refcard.tex \
 	pcb.css \
@@ -119,11 +120,24 @@ else
 	@echo "Skipping documentation extraction since you are not building from GIT sources"
 endif
 
-SUFFIXES = .pcb .pdf .png .tab .tex
+SUFFIXES = .eps .pcb .pdf .png .tab .tex
+
+.pcb.eps :
+	${PCB} -x eps --only-visible --font-path $(top_srcdir)/src --eps-file $@ $<
 
 .pcb.png :
 	${PCB} -x png --only-visible --font-path $(top_srcdir)/src --outfile $@ $<
 
+.eps.pdf :
+if MISSING_PS2PDF
+	@echo "****************************************************"
+	@echo "WARNING:  ps2pdf is missing on your system but"
+	@echo "$@ is out of date and needs to rebuilt."
+	@echo "Changes to $< will be ignored"
+else
+	${PS2PDF} `${AWK} 'BEGIN{s=8}; /BoundingBox/ {printf("-r%d -g%dx%d", s*72, s*$$4, s*$$5);}' $<` $< $@
+endif
+
 .tab.texi:
 	${ASCII2TEXI} $< > $@
 
@@ -138,3 +152,6 @@ else
 	${PDFLATEX} $<
 	${PDFLATEX} $<
 endif
+
+.eps.png:
+	${PERL} eps2png --png $< > $@
